packages/hackathon/src/Hackathon.tsx

```diff
@@ -63,9 +62,7 @@ const banner = (currentHackathon: any, hacker?: IHackerProps) => {
 }
 
 export const Hackathon: FC<HackathonProps> = hot(() => {
-  const extSdk = getExtensionSDK()
-  const adaptor = new ExtensionAdaptor(extSdk)
-  const themeOverrides = adaptor.themeOverrides()
+  const themeOverrides = getThemeOverrides(true)
```

Comment:
So we don't have to fix this later, this should use the `hostedInternally()` check